svf-llvm/lib/SVFIRBuilder.cpp

```diff
@@ -743,7 +743,19 @@ void SVFIRBuilder::visitCastInst(CastInst &inst)
             opnd = stripAllCasts(opnd);
 
         NodeID src = getValueNode(opnd);
-        addCopyEdge(src, dst);
+        // if ZEXT
+        if(SVFUtil::isa<llvm::ZExtInst>(&inst))
```

Comment:
try to find all other invocation sites that calls addCopyEdge., like processCE, InitialGlobal, visitVAArgInst, visitFreezeInst, handleExtCall